src/coreclr/hosts/corerun/corerun.cpp

```diff
@@ -33,7 +33,7 @@ static const wchar_t *coreCLRDll = W("CoreCLR.dll");
 static const wchar_t *coreCLRInstallDirectory = W("%windir%\\system32\\");
 
 // Encapsulates the environment that CoreCLR will run in, including the TPALIST
-class HostEnvironment 
```

Comment:
This file only contains whitespace changes and isn't related to the PR. It should probably be dropped.

Reply:
Yes, it is my workflow error, as I use DebugBreak() in corerun.cpp to start debugging and my editor automatically corrects all whitespace errors. Just forgot to undo changes before commit. Will revert.